<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPredicitionWgtClass</class>
 <widget class="QWidget" name="CPredicitionWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPredicitionWgt</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>辐射查询参数</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>高级查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>图形属性设置</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>类型选择</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>电场选择</string>
       </property>
      </item>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>数据</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <column>
             <property name="text">
              <string>日期</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>实测功率(KW)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>日前预测功率(KW)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>当日预测功率(KW)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>超短期预测功率(KW)</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>导出</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>图形</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="predicition_wgt.qrc"/>
 </resources>
 <connections/>
</ui>
